### ISSUE TITLE
welcome software developers from international community (aka onboarding)

### ISSUE THREAD

**Issue by dckc** [rchain/bounties#295]
following up on introductions (#222) and onboarding (#15, #253)...

Welcome aboard, everybody! Some of you are productively engaged in the RAM process; for the rest of you, here's hoping!

Note that RChain has a core dev team nearby; we can and do aim to help them out, but they have different processes. See #184 and/or contact @MParlikar .

@AbnerZheng did some good work on VM primitives (#176). I helped him navigate the budget and reward process (see CONTRIBUTING.md).

I chatted with @cenyuhai and answered some of his questions.

@zsluedem included Software Development in his skills inventory (#222), but he seems most active in Chinese translation so far. Likewise @david405 (#280 portuguese).

@as1ndu included Software Development in his skills inventory; his github profile shows him from Kampala, Uganda. I have yet to get in contact with you, @as1ndu but if you have any questions, I'm around. (@dckc on github and in discord)

Likewise hello @vadimtrifonov in Tallinn, Estonia and @nzpr (_who shared a location in our survey but not in a public place than I can find_).

¡Hola @msicilia! Much gusto. (Yo hablo un poco.) His [linkedin profile](https://www.linkedin.com/in/msicilia/) shows his location as Madrid, Spain. :heart: 

In the introduction survey (#222) a few people have given only email addresses; here's hoping for time to reach out to them (or someone to do with with me!).

_I should perhaps split out a separate issue for onboarding each person, but not just now..._

**Comment by dckc** [comment]
@patrick727 the "Developer Brain Picking Session"... do you want to share any of your notes here?

**Comment by JieL314** [comment]
I am a people who can only compile some simple programs with c++ or python, but I have a great passion for the blockchain. 
Can I engage in software development or testing？

**Comment by zsluedem** [comment]
That's really inspiring for me . I am kind of flattered, actually.

**Comment by dckc** [comment]
@JieL314 one [guidelines](https://github.com/rchain/Members/wiki/Welcome-To-RAM) from @pmoorman's experience joining the community is:

> **Be a self-starter** Being a RAM is much more like being an _entrepreneur_ than it is like being a _employee_.

That is: rather than ask "can I engage?" we advise that you go ahead and invest some of your time and see if you get rewarded.

Meanwhile, your question reminds me of something I discussed with @TrenchFloat : review of the recently release Rholang tutorial by casual developers. See #301 .

As to testing, yes, of course:

> We invite all community members to test the code, use Rholang and provide feedback and ask questions.
>
> - Questions & Comments [via our Gitter channel](https://gitter.im/rchain/Releases).
> - [Bugs can be filed in our Jira.](https://goo.gl/EmAAjk)

 - [RChain Rholang SDK (v0.1) Released](https://medium.com/rchain-cooperative/sdk-v0-1-5f6bc35bf3e9) Jan 31
 - [RChain Node.Hello (v0.1) Released](https://medium.com/rchain-cooperative/rchain-node-hello-v0-1-released-98d255a9567d) Dec 27, 2017

**Comment by cenyuhai** [comment]
@dckc Can you assign a task for me?

**Comment by dckc** [comment]
@cenyuhai how about [ROS-253][]? That's in the core dev team's [bounties][] list (from #176) and unclaimed. As I mentioned, @AbnerZheng completed a similar task; you can model your work after his (see [PR 121][]) and perhaps chat with him about it.

p.s. I just tried to explain that RAMs are expected to be **self-starters**. "Can you assign a task for me?" is very similar to "can I engage?" which is not self-starter style. If you choose something to work on and get stuck, I'm happy to help - even if you're stuck getting started. But as the [guidelines][] say, _there is no boss_ here; please don't expect me to choose tasks for you.

[ROS-253]: https://rchain.atlassian.net/browse/ROS-253
[bounties]: https://rchain.atlassian.net/issues/?jql=labels%20%3D%20bounty
[guidelines]: https://github.com/rchain/Members/wiki/Welcome-To-RAM
[PR 121]: https://github.com/rchain/rchain/pull/121

**Comment by dckc** [comment]
@David405 and I had a good chat. He gave me a better link to [his profile](https://www.linkedin.com/in/david-asamonye-171454b0/) where I learned that he is part of the **tech startup community in Africa**.

(The profile link he gave in the introductions survey (#222) was broken somehow.)

I used part of the [architecture document](http://rchain-architecture.readthedocs.io/en/latest/) to answer one of his questions (about what operating systems rholang runs on; more on that in #298) and I discovered that he hadn't read it yet. So many of you are involved in translating things like the architecture document that I had started to assume everyone had read it. Not a good assumption.

Anyone who hasn't followed the link from https://www.rchain.coop/ to [FAQ](https://github.com/rchain/reference/blob/master/faq.md), I strongly recommend investing a few hours in reading the FAQ and following most of the pointers from it.
